src/extensions/extensionsHelper.ts

```diff
@@ -108,9 +108,12 @@ export function getDBInstanceFromURL(databaseUrl = ""): string {
  * Gets Firebase project specific param values.
  */
 export async function getFirebaseProjectParams(
-  projectId: string,
+  projectId: string | undefined,
```

Comment:
gave this a try but can't because emulator mode has a default, and having 2 optional positional arguments is weird, could change to named param but then we lose the default :/  so I think I'll just leave this as is